content/_releases/22/22.md

```diff
@@ -0,0 +1,235 @@
+*   Fixed the user interface bug connected with the Dashboard graph when the horizontal line and values would sometimes overlap.
+*   Addressed the problem when the registration page would sometimes create an empty contract even if the user would already exist in the Database.
+*   Fixed the user interface bug connected with the loading a flow versions page when the aggregated document size would exceed 16MB limit of MongoDB.
+
+## Components
+
+### [REST API V2 component](/components/rest-api/)
+
+*   **IMPROVED** the use of platform secrets
+
+### [HJson component](/components/hjson/) `1.0.2`
+
+*   **FIXED** the attachments URL
+
+### [Delta-detection Component](components/delta-detection/) `2.1.0`
+
+*   **ADDED** `Write Entire Bucket Contents` action
+*   **ADDED** `Read Entire Bucket Contents` action
+
+### [utility component](/components/utility/) `1.4.2`
+
+*   **ADDED** `Network Diagnostics` action
+
+### [Ipaas component](/components/ipaas/) `1.4.0`
+
+*   **FIXED** issue - hydrate flow with `pubSubTopicName` at actions `Lookup Object By Unique Criteria` and `Lookup Objects (plural)`
+*   **FIXED** issue - add logic to handle matching Pub-Sub topics at action `Upsert Object By Unique Criteria`
+*   **FIXED** issue - hydrate flow with `componentSemanticVersion` at actions `Lookup Object By Unique Criteria` and `Lookup Objects (plural)`
+*   **FIXED** issue - add logic to handle matching on Semantic Versions at action `Upsert Object By Unique Criteria`
+
+### [Zoho CRM component](/components/zoho-crm/) `1.3.4`
+
+*   **FIXED** the attachment processing
+*   **FIXED** Set the node engine to `16.x` (addresses error with Verify Credentials)
+*   **ADDED** Environment variable `MAX_FILE_SIZE` to control attachment size limit
+*   **UPDATED** component dependencies
+
+### [Magento2 component](/components/magento2) `1.6.7`
+
+*   **ADDED** support for Magento2 versions `2.4.4`
+*   **FIXED** the integration tests
+
+### Ukraine-alert component `1.0.0`
+
+*   **NEW** component to receive raid alerts notifications in Ukraine
+*   **ADDED** new `Get Ukraine Alerts Polling` trigger
+*   **ADDED** new `Webhook` trigger
+
+### [Google spreadsheet component](/components/gspreadsheet) `2.0.3`
+
+*   **IMPROVED** Moved a delay between calls value from environment variables to credentials
+*   **ADDED** a number of retry attempts field to credentials
+*   **ADDED** retry logic that allows to avoid Quota exceeded errors
+
+### [Docuware component](/components/docuware) `1.1.1`
+
+*   **FIXED** bug with data shown in "Search dialog to use" field
+*   **FIXED** attachments Url
+
+### [JDBC component](/components/jdbc) `2.5.5`
+
+*   **UPDATED** connection parameter `connectionTimeZone`, now it defaults to `SERVER` for MySQL connections
+
+### [AWS S3 component](/components/aws-s3)
+
+*   **FIXED** bug in Get New and Updated S3 Objects trigger when selected Emit Behaviour - Fetch All
```

Comment:
bug in 'Get New and Updated S3 Objects' trigger when the 'Emit Behaviour' - 'Fetch All' is selected

(I think it will be helpful to harmonise style where we describe: Pages/views of the UI; Field names; Variables)